modules/Auth/lib/AuthHandler.cpp

```diff
@@ -161,6 +176,7 @@ TokenHandlingResult AuthHandler::handle_token(const ProvidedIdToken& provided_to
         while (i < validation_results.size() && !authorized && !referenced_connectors.empty()) {
             auto validation_result = validation_results.at(i);
             if (validation_result.authorization_status == AuthorizationStatus::Accepted) {
+                this->publish_token_validation_status_callback(provided_token, types::authorization::TokenValidationStatus::Accepted);
```

Comment:
Just a note:
I can understand why the status is published here because a validator has accepted it, but its important to note that no authorization has yet been provided to any EvseManager at this point. The `provided_token` might contain references to multiple connectors (== EvseManager(s)) so publishing a status with `Accepted` here doesn't mean that a session/transaction can be started at these connectors. 

Authorization is finally only provided when a connector was selected and the `notify_evse_callback` is executed.

Reply:
Yes, we are aware of that